packages/next/lib/load-env-config.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path'
+import dotenv from 'next/dist/compiled/dotenv'
+import { ENV_FILE } from '../next-server/lib/constants'
+import dotenvExpand from 'next/dist/compiled/dotenv-expand'
+
+export type Env = { [key: string]: string }
+
+export function loadEnvConfig(dir: string, dev?: boolean): void {
+  const isTest = process.env.NODE_ENV === 'test'
+  const mode = isTest ? 'test' : dev ? 'development' : 'production'
+  const dotenvFiles = [
+    `${ENV_FILE}.${mode}.local`,
+    `${ENV_FILE}.${mode}`,
+    // Don't include `.env.local` for `test` environment
+    // since normally you expect tests to produce the same
+    // results for everyone
+    mode !== 'test' && `${ENV_FILE}.local`,
+    ENV_FILE,
+  ].filter(Boolean) as string[]
+
+  for (const envFile of dotenvFiles) {
+    // only load .env if the user provided has an env config file
+    const dotEnvPath = path.join(dir, envFile)
+    const result = dotenvExpand(dotenv.config({ path: dotEnvPath }))
+
+    if (result.parsed) {
+      console.log(`Loaded env from ${envFile}`)
```

Comment:
Let's jazz this message up a bit and make it match our Babel one.

```js
            console.log(`> Using external babel configuration`)
            console.log(`> Location: "${file}"`)
```

Maybe something like:
```suggestion
      console.log(`> ${chalk.cyan.bold('Info:')} Loaded env from ${envFile}`)
```